interop-testing/src/main/java/io/grpc/testing/integration/XdsTestClient.java

```diff
@@ -281,6 +280,15 @@ private void makeRpc(final RpcType rpcType, final Metadata headersToSend) {
           currentRequestId += 1;
           requestId = currentRequestId;
           savedWatchers.addAll(watchers);
+          if (!rpcsStartedByMethod.containsKey(rpcType.name())) {
```

Comment:
nit/optional: is this a standard way of working around Java 7's lack of `getOrDefault`? This seems slightly "tricky" to me in that it requires thinking to see that that value in the map later on will never be null when `rpcsSucceededByMethod.put(rpcType.name(), succeededBase + 1);`-type code is executing. Correctness here also requires knowing that no key is ever removed from these maps, which could be less obvious in the future.
 
I would prefer dropping this block here and just doing the null check when incrementing the value each time, as it seems a bit clearer for future readers:

```
int rpcsStarted = 0;
if (rpcsStartedBymethod.containsKey(rpcType.name()) {
    rpcsStarted = rpcsStartedByMethod.get(rpcType.name());
}
rpcsStartedByMethod.put(rpcType.name(), rpcsStarted + 1);
```

But then we have to use the ugly block above in several places as well, so will let you choose whether it's better as-is versus my proposal.

Reply:
Yeah, Java 7 is cumbersome for updating map value. Updated to what you suggested.